crates/biome_analyze/src/lib.rs

```diff
@@ -670,6 +721,8 @@ pub enum AnalyzerSuppressionKind<'a> {
     Rule(&'a str),
     /// A suppression to be evaluated by a specific rule eg. `// biome-ignore lint/correctness/useExhaustiveDependencies(foo)`
     RuleInstance(&'a str, &'a str),
+    /// A suppression disabling a plugin eg. `// biome-ignore plugin/my-plugin`
```

Comment:
```suggestion
    /// A suppression disabling a plugin eg. `// lint/biome-ignore plugin/my-plugin`
```

Reply:
@arendjr Isn't it `biome-ignore lint/plugin/my-plugin`?